cms/bundles/models.py

```diff
@@ -26,6 +29,8 @@
 
     from cms.teams.models import Team
 
+PREVIEWER_EXCLUDED_PAGE_TYPES = (HomePage, TopicPage, ReleaseCalendarPage)
```

Comment:
Given the updated requirements on CMS-471:
> Business Areas should be able to preview the linked release calendar page as it would look like when published

I think it makes sense to allow previewing release calendar pages when they are added as bundle content

Reply:
Removed in [05bd5ac](https://github.com/ONSdigital/dis-wagtail/pull/183/commits/05bd5ac3a07aa56c0fe678cd3242d8620098a0b8)

Reply:
`test_inspect_view__previewers__contains_only_relevant_pages` now needs updating